reference/carts.sf.yml

```diff
@@ -817,6 +820,9 @@ components:
         listPrice:
           type: number
           description: The net item price before discounts and coupons. It is based on the product default price or sale price (if set) configured in BigCommerce Admin.
+        originalPrice:
+          type: number
+          description: 'Item’s original price, set as product’s default price in the admin panel'
```

Comment:
```suggestion
          description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

Reply:
Thanks for the feedback, but for this original price, it's actually prodduct's default price which already set in admin panel, and here it's just return that value in response. if begin with `Set ...`, i think they might misunderstand that it tell them to set the price, but they don't need.